src/python/aqueduct_executor/operators/connectors/data/config.py

```diff
@@ -54,8 +54,11 @@ class S3Config(models.BaseConfig):
     config_file_profile: str = ""
 
     bucket: str = ""
-
     region: str = ""
+
+    # Expected to be in the format "path/to/dir/" (without a leading slash, but with a trailing one).
+    root_dir: str = ""
```

Comment:
Is this used in data connection? I thought it's only relevant for S3 used as storage.

Reply:
It has to be here because the model forbids extra fields. I'll add a note that it's actually unused.

Reply:
reminder to add the note.